src/Symfony/Component/FeatureFlag/FeatureChecker.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureFlag;
+
+use Symfony\Component\FeatureFlag\Provider\ProviderInterface;
+
+final class FeatureChecker implements FeatureCheckerInterface
```

Comment:
shouldnt you comment `@experimental` here ?

Reply:
Sure! Good catch.